content/building/cmake-presets.md

```diff
@@ -4,6 +4,123 @@
 
 This document describes how to use and modify the **CMakePresets.json** and **CMakeUserPresets.TEMPLATE.json** files to suit your needs.
 
-## DOCUMENTATION
+## Build options for CMake Presets
 
-Please refer to [this blog post](https://www.nanoframework.net/build-updated-to-cmake-presets/) and some useful information from the now **obsolete**  [cmake-tools-cmake-variants](cmake-tools-cmake-variants.md) until it is fixed.
+Follows a list of the build options to use in the CMake presets.
```

Comment:
```suggestion
Follows is a list of build options you can use for target boards in CMake Presets.
```

Reply:
It depends on what you are trying to convey. Perhaps it would be better as `Below is a list of ...` or `The following is a list of...`

Reply:
Given that non native speakers need to understand it 😅

Reply:
I'm not debating this one for sure! Feel free to change to whatever makes more sense to you.

Reply:
Also, given this list will likely become outdated, we should possibily use the words "example"...

Reply:
This is not an example. It's meant to be an exhaustive list of the available build options that NF build system "understands"

Reply:
Ok, but as we all know, docs fall behind, so in that case, perhaps we should specify "Correct as of", and/or link to the current cmake file...

Reply:
Yes, docs fall behind. But I don't think this is the point... this doc is meant to collect all the available build options. 
Some may even not be present in the various cmake presets because they are not being used. Therefore, I don't know how we can easily link to all cmake files where those live... 😯 🤔